HerPublicWebsite/ExternalServices/GoogleAnalytics/GoogleAnalyticsService.cs

```diff
@@ -79,7 +79,7 @@ private async Task SendEvent(GaRequestBody body)
     // If we can't find the _ga cookie, return a new id
     private string GetClientId(HttpRequest request)
     {
-        return cookieService.TryGetCookie<string>(request, Configuration.CookieName, out var cookie) 
+        return request.Cookies.TryGetValue(Configuration.CookieName, out var cookie)
```

Comment:
The `TryGetCookie` method tries to parse the cookie as JSON, but we actually just want the raw cookie value (as a string), so don't need the helper.

Reply:
I assume you've tested this and it just works :P